Gruntfile.js

```diff
@@ -63,16 +83,28 @@ module.exports = function(grunt) {
         watch: {
             js: {
                 files: ['js/**/*.js'],
-                tasks: ['concurrent', 'concat', 'uglify']
+                tasks: ['jslint', 'concat', 'uglify']
+            },
+            css: {
+                files: ['css/**/*.scss'],
+                tasks: ['scsslint', 'sass']
             }
         },
 
         concurrent: {
-            lint: ['jshint', 'jscs']
+            jslint: ['jslint'],
+            js: [['concurrent:jslint', 'concat', 'uglify']],
```

Comment:
Now I see the reason, interesting point. I never used this approach.

I have to say that, personally, I don't really like as it's an abuse of concurrent. As we discussed, if you wish to run tasks in sequence, then you should have them outside of concurrent. After all, the whole purpose of concurrent is to run tasks concurrently.